woocommerce/woocommerce
# update e2e dependencies to be consistent with main package

### Changes proposed in this Pull Request:

The main package dependencies were being updated through `renovatebot`. `renovatebot` didn't look at the E2E packages for corresponding dependencies. This PR bumps the E2E dependency versions to match where possible.

### How to test the changes in this Pull Request:

1. Verify CI
2. Run `npm i` locally
3. Run E2E tests locally

### Changelog entry

> N/A
